src/clarity-icons/shapes/essential-shapes.ts

```diff
@@ -1703,13 +1655,29 @@ export const essentialShapes: any = {
         </svg>`
 
 
-
 };
 
+Object.defineProperty(essentialShapes, "edit", descriptorConfig(essentialShapes["pencil"]));
+Object.defineProperty(essentialShapes, "note-edit", descriptorConfig(essentialShapes["note"]));
+Object.defineProperty(essentialShapes, "group", descriptorConfig(essentialShapes["users"]));
+Object.defineProperty(essentialShapes, "ellipses-vertical", descriptorConfig(essentialShapes["ellipsis-vertical"]));
```

Comment:
TODOs should be moved here so we can get rid of `ellipses-vertical` and `ellipses-horizontal`.